inst/include/kde1d/kde1d.hpp

```diff
@@ -600,6 +601,7 @@ Kde1d::fit_lp(const Eigen::VectorXd& x,
   fft::KdeFFT kde_fft(
       x, bandwidth_, grid_points(0), grid_points(m - 1), weights);
   Eigen::VectorXd f0 = kde_fft.kde_drv(0);
+  Eigen::VectorXd f1(f0.size()), f2(f0.size());
```

Comment:
Why do this memory allocation here if not needed?

Reply:
We use both in calculate_infl() below even if `degree = 0`. These are fixed-size (grid length), and the allocation here should be negligible.